src/py/run_generator.rs

```diff
@@ -193,78 +123,66 @@ impl IntoPy<PyObject> for ErrorCode {
             ErrorCode::DuplicateOutput => 4,
             ErrorCode::DoubleSpend => 5,
             ErrorCode::CostExceeded => 23,
+            ErrorCode::RuntimeError => 117,
         };
         ret.to_object(py)
     }
 }
 
-// returns the cost of running the CLVM program along with the list of NPCs for
-// the generator/spend bundle. Each SpendConditionSummary is a coin spend along with its
-// conditions
+// returns the cost of running the CLVM program along with conditions and the list of
+// spends
 #[allow(clippy::too_many_arguments)]
 #[pyfunction]
-pub fn run_generator(
+pub fn run_generator2(
```

Comment:
The reason I didn't keep the previous `run_generator()` around for backwards compatibility was that it would require a fair amount of work (and tests) to convert the new internal data structure to the previous returned structure. It didn't seem worth the effort for not much practical benefit.